packages/launchpad/src/setup/SelectFwOrBundler.vue

```diff
@@ -74,6 +74,12 @@
         v-html="itemValue.icon"
       />
     </template>
+    <template
```

Comment:
The fact we need this check it starting to tell me that this component is a bit too smart... along with the name `SelectFwOrBundler`.

Not going to block this but if you did want to refactor this to have `SelectFramework` and `SelectBundler` which are composed from some shared components, I would not be against it - WDYT?

Again not a blocker, but wouldn't be against doing a bit of clean up here, either.

Reply:
yeah not ideal. I'll look into refactoring it

Reply:
Are we looking to do this as part of this PR?

Reply:
Let's leave it for now 👍🏻